.github/workflows/lint.yml

```diff
@@ -2,24 +2,28 @@
 
 name: Lint
 
-on: [pull_request]
+on: 
+  pull_request:
+    paths-ignore: ['**.md', 'docs/**']
+
+# Ensure that only a single job or workflow using the same
+# concurrency group will run at a time. This would cancel
+# any in-progress jobs in the same github workflow and github
+# ref (e.g. refs/heads/main or refs/pull/<pr_number>/merge).
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
The value of `workflow` is different, so no impact. Deleting this section though, lint is lightweight. I added this when fiddling with clang-tidy in a separate branch, since that takes much longer and should account for concurrency.

> The name of the workflow. For example, My test workflow. If the workflow file doesn't specify a name, the value of this variable is the full path of the workflow file in the repository.